.github/ISSUE_TEMPLATE/housekeeping-request.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Housekeeping request
+about: Suggestions for improving code quality
+title: "[Housekeeping]"
+labels: NOT Prioritized, NOT Tech refined, housekeeping
+assignees: ''
+---
+
+**Short description of housekeeping request**
+
+<!-- Replace the paragraph below with your own description -->
+
+Please provide a clear and concise description of what needs housekeeping and the reason for doing it.
```

Comment:
```suggestion
Add a clear and concise description of what needs housekeeping along with a reason for doing it.
```

While very polite - please is not used in the paragraphs that is meant to be replaced in the other templates 😄